### PR TITLE
Vehicle pickup water Gamecommand

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

File: src/OpenLoco/GameCommands/VehiclePickup.cpp [hunk 1]
```diff
@@ -8,17 +8,20 @@
...
 {
     static loco_global<Utility::prng, 0x00525E20> _prng;
 
     // 0x0048B15B
-    static void playPickupSound(Vehicles::Vehicle2* veh2)
+    void playPickupSound(Vehicles::Vehicle2* veh2)
     {
         const auto frequency = _prng->randNext(20003, 24098);
         Audio::playSound(Audio::SoundId::vehiclePickup, veh2->position, -1000, frequency);
     }
```
[the remark this answers is not in the document]
Comment (duncanspumpkin) [reply src/OpenLoco/GameCommands/VehiclePickup.cpp:20]: Yeah just haven't decided where

**Review by AaronVanGeffen — APPROVED** [review]
Appears to be working fine.

**Comment by duncanspumpkin** [comment]
I want to apply my vehicle visitor to this so don't merge until that is done

**Review by AaronVanGeffen — APPROVED** [review]
The use of the vehicle visitor pattern cleans the code up nicely! Still appears to be working fine, too.

**Pull request merged**